clrdefinitions.cmake

```diff
@@ -122,6 +122,9 @@ endif()
 if(FEATURE_GDBJIT_FRAME)
     add_definitions(-DFEATURE_GDBJIT_FRAME)
 endif(FEATURE_GDBJIT_FRAME)
+if(FEATURE_GDBJIT_LANGID)
```

Comment:
A nit - all of our FEATURE_XXX definitions (both in CMake and in C++) are value-less. So adding a new one that has a value feels a bit out of line. I would prefer using FEATURE_GDBJIT_LANGID_CS and use it to select whether we embed the DW_LANG_C89 or your new CS langid.

Reply:
Done. Thank you!